Content.Shared/Clumsy/ClumsySystem.cs

```diff
@@ -97,14 +98,17 @@ private void BeforeGunShotEvent(Entity<ClumsyComponent> ent, ref SelfBeforeGunSh
     private void OnBeforeClimbEvent(Entity<ClumsyComponent> ent, ref SelfBeforeClimbEvent args)
     {
         // checks if ClumsyVaulting is false, if so, skips.
-        if (!ent.Comp.ClumsyVaulting)
+        if (!ent.Comp.ClumsyVaulting
+            || HasComp<BiomassReclaimerComponent>(args.BeingClimbedOn))
```

Comment:
Yeah, I'm definitely not a fan of hard coding this component check, as slarticodefast said. Is this check even needed with the other check in place?

Reply:
We probably don't even need to use `SelfBeforeClimbEvent` here and can use normal drag drop without any climbing for inserting into the reclaimer, similar ot the cryopod.